core/ansi_escape_parser.js

```diff
@@ -21,10 +21,16 @@ function ANSIEscapeParser(options) {
     events.EventEmitter.call(this);
 
     this.column             = 1;
-    this.row                = 1;
     this.scrollBack         = 0;
     this.graphicRendition   = {};
 
+    if(options.startRow != null) {
```

Comment:
Careful with `null` here due to JS `undefined`

Reply:
Thanks I'll change this.

Reply:
Fixed.

(I believe in the original it should have already been checking for both null and undefined since null == undefined, but it is a good call this makes it more explicit.